cumulusci/tasks/marketing_cloud/tests/test_get_user_info.py

```diff
@@ -0,0 +1,130 @@
+            "email": "example@example.com",
+            "locale": "en-GB",
+            "zoneinfo": "Europe/London",
+            "timezone": {
+                "longName": "(GMT) Dublin, Edinburgh, Lisbon, London *",
+                "shortName": "GMT+0",
+                "offset": 0,
+                "dst": True,
+            },
+        },
+        "organization": {
+            "member_id": 10123456,
+            "enterprise_id": 10123456,
+            "enterprise_name": "Auth_enterprise_name",
+            "account_type": "enterprise",
+            "stack_key": "S1",
+            "region": "NA1",
+            "locale": "en-US",
+            "zoneinfo": "America/Los_Angeles",
+            "timezone": {
+                "longName": "(GMT-08:00) Pacific Time (US & Canada) *",
+                "shortName": "GMT-8",
+                "offset": -8,
+                "dst": True,
+            },
+        },
+        "rest": {
+            "rest_instance_url": "https://mc563885gzs27c5t9-63k636ttgm.rest.marketingcloudapis.com",
+            "soap_instance_url": "https://mc563885gzs27c5t9-63k636ttgm.soap.marketingcloudapis.com",
+        },
+        "application": {
+            "id": "1a23b4cd-5e66-789f-0g1h-2i3a6efb6d80",
+            "name": "auth_application_name",
+            "redirectUrl": [
+                "https://example.example.com*",
+                "https://example.com/oauth-authorize",
+                "https://example.example.com/***********",
+            ],
+            "appScopes": [
+                "openid",
+                "offline",
+                "email_read",
+                "email_send",
+                "email_write",
+            ],
+        },
+        "permissions": [
+            {
+                "objectTypeName": "Email",
+                "operationName": "Update",
+                "name": "Update",
+                "id": 123,
+            }
+        ],
+    }
+
+
+@responses.activate
+def test_get_user_info__success(get_user_info_task, user_info_payload):
+    responses.add(
+        "POST",
+        "https://tssd.auth.marketingcloudapis.com/v2/token",
+        json={"access_token": "ACCESS_TOKEN"},
+    )
+    responses.add(
+        "GET",
+        f"https://TSSD.auth.marketingcloudapis.com/{MC_API_VERSION}/userinfo",
+        json=user_info_payload,
+    )
+
+    get_user_info_task()
+
+    expected_payload = user_info_payload
+    del expected_payload["rest"]
+    del expected_payload["application"]
+
+    assert get_user_info_task.return_values == expected_payload
+
+
+@responses.activate
+def test_get_user_info__HTTPError(get_user_info_task, user_info_payload):
+    responses.add(
+        "POST",
+        "https://tssd.auth.marketingcloudapis.com/v2/token",
+        json={"access_token": "ACCESS_TOKEN"},
+    )
+    responses.add(
+        "GET",
+        f"https://TSSD.auth.marketingcloudapis.com/{MC_API_VERSION}/userinfo",
+        body=requests.exceptions.HTTPError("bad request"),
```

Comment:
If you're trying to simulate a 400 Bad Request response, I think you want status=400 and body= a string, not an error instance